tgui/packages/tgui/interfaces/CryoCell.js

```diff
@@ -0,0 +1,224 @@
+/**
+ * @file
+ * @copyright 2023
+ * @author Garash (https://github.com/Garash2k)
+ * @license ISC
+ */
+
+import { useBackend } from "../backend";
+import { Button, Box, Icon, AnimatedNumber, Section, LabeledList, ProgressBar, Dimmer, Table } from "../components";
+import { Window } from '../layouts';
+import { getTemperatureColor, getTemperatureIcon } from './common/temperatureUtils';
+import { ReagentGraph, ReagentList } from './common/ReagentInfo';
+import { HealthStat } from './common/HealthStat';
+import { DisplayBloodPressure, DisplayTempImplantRow, DisplayRads, DisplayBrain, DisplayEmbeddedObjects } from '../interfaces/OperatingComputer/index';
```

Comment:
@mordent-goonstation should these be in common?

Reply:
Refactored to common/KeyHealthIndicators